HerPublicWebsite.BusinessLogic/ExternalServices/S3FileWriter/S3FileWriter.cs

```diff
@@ -34,11 +34,8 @@ public async Task WriteFileAsync(string custodianCode, int month, int year, Stre
         
         try
         {
-            // TODO: BEISHER-240 do we need a username/password at all - or is the user this is running as get given permissions externally?
-            // https://docs.aws.amazon.com/sdk-for-net/v3/developer-guide/creds-assign.html
-            // https://stackoverflow.com/questions/47124876/how-to-specify-aws-credentials-in-c-sharp-net-core-console-program
```

Comment:
We definitely want to use a role here, rather than passing in credentials. The way this works for ECS is you configure an 'ECS Task Execution Role'* with the permissions you want.

In local development, you can set this up by configuring credentials in `~/.aws/config` / `~/.aws/credentials`, or configuring the corresponding environment variables. If we do lots of dev we might want to set up a filesystem implementation of the file writer instead, but not critical.

*Not to be confused with an 'ECS Task Role', which is the role used to create the tasks themselves (so needs permissions to create new tasks and pull the docker images etc.)